hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotCreateRequest.java

```diff
@@ -341,12 +341,8 @@ private long estimateBucketDataSize(OmBucketInfo bucketInfo) {
     if (defRC == null) {
       // Note: A lot of tests are not setting bucket DefaultReplicationConfig,
       //  sometimes intentionally.
-      //  Fall back to config default and print warning level log.
+      //  Fall back to config default.
       rc = ReplicationConfig.getDefault(new OzoneConfiguration());
```

Comment:
Suggestion for additional improvement: let's also pass `ozoneManager.getDefaultReplicationConfig()` to `estimateBucketDataSize` and use it instead of creating a new config here.